kolibri/plugins/management/assets/src/management-styles.styl

```diff
@@ -1,15 +0,0 @@
-  @require '~kolibri.styles.definitions'
-
-  .manage-content
-    width: 100%
-    @media screen and (max-width: $medium-breakpoint)
-        width: 90%
-        margin-left: auto
-        margin-right: auto
-
-  .page
```

Comment:
Some of this style should have not been removed.

Reply:
Are you sure? I couldn't actually find a reference to the `page` class

Reply:
oh, I found it here .. https://github.com/learningequality/kolibri/pull/2215/files#diff-a895b0f385c04308a48a2128fbbbb789L9

I'll check the styling again. It might actually be good to remove these styles, because they should be set globally? It's not clear to me why the background color would be different for this one section

Reply:
added back in